ebs/ebs_service.go

```diff
@@ -663,16 +663,10 @@ func (s *ebsService) GetMostRecentAvailableVolume(volumeName string, dcName stri
 					aws.String(dcName),
 				},
 			},
-			&ec2.Filter{
```

Comment:
@jaytaylor Because now we are checking if the most recent volume is available later. Basically, if a mesos-slave is lost and the job is relocated. We want it to err out and say that its in use so we can quickly track it down. Also, in a multi-az scenario, if the job moves to another AZ then there very well could be an available volume in the other AZ and then it will snapshot and create from that causing data loss